metricbeat/module/mongodb/status/data.go

```diff
@@ -0,0 +1,241 @@
+package status
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+func eventMapping(status map[string]interface{}) common.MapStr {
+	errs := map[string]error{}
+	event := common.MapStr{
+		"version": mustBeString("version", status, errs),
+		"uptime": common.MapStr{
+			"ms": mustBeInteger("uptimeMillis", status, errs),
+		},
+		"local_time":         mustBeTime("localTime", status, errs),
+		"write_backs_queued": mustBeBool("writeBacksQueued", status, errs),
+	}
+	removeErroredKeys(event, errs)
```

Comment:
The current approach was to set empty / default values for non existing values to not "break" dashboards / queries because of missing values. But this approach could be even more powerful and better cross version compatible.